bip-PC.md

```diff
@@ -0,0 +1,187 @@
+<pre>
+  BIP: ?
+  Layer: Consensus (soft fork)
+  Title: OP_PAIRCOMMIT
+  Author: moonsettler <moonsettler@protonmail.com>
+  Comments-Summary: No comments yet.
+  Comments-URI: <links to wiki page for comments>
+  Status: Draft
+  Type: Standards Track
+  Created: 2024-11-08
+  License: BSD-3-CLAUSE
+</pre>
+
+## Abstract
+
+This BIP describes a new tapscript opcode `OP_PAIRCOMMIT` which
+provides limited vector commitment functionality in tapscript.
+
+When evaluated, the `OP_PAIRCOMMIT` instruction:
+* Pops the top two values off the stack,
+* takes the "PairCommit" tagged SHA256 hash of the stack elements,
```

Comment:
consider linking to the section in bip340 about tagged hashes

Reply:
It is linked from Specification (line 53), not sure I would want to link from here.